src/test/java/seedu/address/storage/JsonSerializableTutorsPetTest.java

```diff
@@ -25,7 +26,11 @@ public void toModelType_typicalStudentsFile_success() throws Exception {
         JsonSerializableTutorsPet dataFromFile = JsonUtil.readJsonFile(TYPICAL_STUDENTS_FILE,
                 JsonSerializableTutorsPet.class).get();
         TutorsPet tutorsPetFromFile = dataFromFile.toModelType();
-        TutorsPet typicalStudentsTutorsPet = TypicalStudent.getTypicalTutorsPet();
+        TutorsPet typicalStudentsTutorsPet = TypicalTutorsPet.getTypicalTutorsPet();
+
+        // Workaround as storage functionality for ModuleClasses has not been implemented.
```

Comment:
Comments start with lower letter + no full stops
```suggestion
        // workaround as storage functionality for ModuleClasses has not been implemented
```